Go: iOS support docs (#598)
breez/spark-sdk: docs/breez-sdk/src/guide/install_go.md

```console
go get github.com/breez/breez-sdk-spark-go
```

## Integration

> For [Android](#android) and [Windows](#windows) the provided binding libraries need to be copied into a location where they need to be found during runtime. 

Reviewer: ```suggestion
For [Android](#android) and [Windows](#windows), the provided binding libraries must be copied to a location that can be found at runtime. 
```
